### ISSUE TITLE
Equalizer wont save values.after changing song goes back to flat 

### ISSUE THREAD

**Issue by flascget** [smplayer-dev/smplayer#652]
**Is your feature request related to a problem? Please describe.**

**Describe the solution you'd like**

**Describe alternatives you've considered**

**Additional context**

**Comment by flascget** [comment]
it doesn t save custom value for the equalizer after changing song
so if im ghanging the audio file sould be with the same custom equaliser custom settings ,instead it goes to flat

**Comment by smplayer-dev** [comment]
After adjusting the equalizer press the button `Set as default values`.

**Comment by flascget** [comment]
yes ,but after justin and pressing the button save it changes after the
next song( that s why i m asking,is it possible to keep changes applied
after the next song? tx.

În dum., 19 feb. 2023 la 17:10, Ricardo ***@***.***> a scris:

**Comment by flascget** [comment]
i did,but after pressing the button Set as default once i change the song the Set goes back to hearing "flat"even tho it shoes the Set on the equalizer,more so ,if i pot the same song previously saved as i was setting the custom Set goes to hearing flat
